### PR TITLE
Fix #2352: Update Ktlint version to "0.37.1"

### LINKED ISSUE #2352: Update Ktlint version from "0.37.0" to "0.37.1"
Error running Ktlint vesion "0.37.0" for android files on Windows 10 OS.

**To Reproduce**
1. Download the ktlint version - "0.37.0" using Git Bash terminal on Windows with the instructions: https://github.com/pinterest/ktlint#installation 
2. After installation, run the commands here [https://github.com/oppia/oppia-android/wiki/Ktlint-Guide#commands](url) 
3. Observe the following error when running on android files:
Reference: https://github.com/pinterest/ktlint/issues/761#issue-630042183 

image

**Expected behavior**
Following image shows the expected results from the linter, (Klinter version - "0.37.1"):
image

**Additional context**
We can upgrade the Ktlint version from "0.37.0" to "0.37.1" to fix this issue. (Results in screenshot above)

### PR DESCRIPTION
**Pull request by srikanthkb**
## Explanation

Fixes #2352

Updated the Ktlint download version in github workflows from "0.37.0" to 0.37.1"

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

### CODE REVIEW

File: .github/workflows/main.yml [hunk 1]
```diff
@@ -27,7 +27,7 @@ jobs:
 
     - name: Download Ktlint
       run: |
-        KTLINT="0.37.0"
+        KTLINT="0.40.0"
```
Comment (FareesHussain) [inline .github/workflows/main.yml]: As there are few errors with this version can you try 0.37.1

as it has a windows fix in that version

**Review by FareesHussain — CHANGES_REQUESTED** [review]
@Srikanth-Kb PTAL

File: .github/workflows/main.yml (hunk 1 above)
Comment (FareesHussain) [reply .github/workflows/main.yml]: reference - https://github.com/pinterest/ktlint/issues/761#issue-630042183

Comment (srikanthkb) [reply .github/workflows/main.yml]: @FareesHussain Thank you for the suggestion, I will confirm with 0.37.1 and update.

**Review by anandwana001 — APPROVED** [review]
LGTM, Thanks @Srikanth-Kb 
Also, please update the issue and pr title for 0.37.1

**Review by FareesHussain — APPROVED** [review]
LGTM, thanks

File: .github/workflows/main.yml (hunk 1 above)
Comment (anandwana001) [reply .github/workflows/main.yml]: resolving on behalf of @FareesHussain

**Comment by srikanthkb** [comment]
> LGTM, Thanks @Srikanth-Kb
> Also, please update the issue and pr title for 0.37.1

**Comment by srikanthkb** [comment]
> LGTM, Thanks @Srikanth-Kb
> Also, please update the issue and pr title for 0.37.1

**Comment by srikanthkb** [comment]
Done.

**Review by rt4914 — APPROVED** [review]
LGTM, thanks.

**Pull request merged**